homeassistant/components/whirlpool/config_flow.py

```diff
@@ -0,0 +1,72 @@
+"""Config flow for Whirlpool Sixth Sense integration."""
+import logging
+
+import aiohttp
+import voluptuous as vol
+from whirlpool.auth import Auth
+
+from homeassistant import config_entries, core, exceptions
+
+from .const import DOMAIN  # pylint: disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema({"username": str, "password": str})
```

Comment:
You should use home-assistant constants for `username` and `password`: https://github.com/home-assistant/core/blob/dev/homeassistant/const.py#L191